setup.py

```diff
@@ -61,6 +51,8 @@
     "distributed>=2.0",
 ]
 
+gis_requirements = ["rtree>=0.9", "clisops>=0.3"]
```

Comment:
I might have missed something, why are we needing rtree?

Reply:
Okay, so this is an issue that I think needs to be addressed in clisops, but essentially, depending on how GeoPandas is installed (via pip or via conda), `rtree` is not guaranteed to be installed alongside it. This might explain some errors in `clisops` (I'm opening a PR to verify this right now) and, assuming this is the case, `rtree` should 100% be listed as a build dependency in that package, not here. Once I've verified this, I can remove this from here. `clisops` should be managing all its dependencies. Nice catch!

The apt package `libspatialindex[-dev]` is configured as a C-lib dependency in order to satisfy this package, hence why I've removed it from the `py36-nosubset` build.

Reply:
This is fixed in `clisops@v0.3.1`